pkg/route/registry/route/strategy.go

```diff
@@ -106,7 +106,7 @@ func (routeStrategy) ValidateUpdate(ctx kapi.Context, obj, old runtime.Object) f
 }
 
 func (routeStrategy) AllowUnconditionalUpdate() bool {
-	return true
+	return false
```

Comment:
`true` here allows stomping the object without setting `resourceVersion`, with `false` you are required to set `resourceVersion` to current value. This protects you from accidentally overriding objects. We have that set to `false` to majority of our resources.